### ISSUE TITLE
Error when clicking on "à un commerçant" or "à un particulier"

### ISSUE THREAD

**Issue by MatSarr** [Lokavaluto/lokavaluto-mobile-application#6]
Phone: oneplus two (rom lineage os 17) android 10

Error: 'Cannot read property "forEach" of undefined'

when clicking on "à un commerçant" or "à un particulier" from "payer mes achats" or "envoyer"

error doesn't obstruct to find a person or a company

**Comment by farfromrefug** [comment]
@MatSarr should be working in version 6 available now

**Comment by MatSarr** [comment]
Yes but now there's an another error also when clicking on "à un commerçant" or "à un particulier" from "payer mes achats" or "envoyer"

Error: Connot convert undefined or null to object

App version: 1.0.0.10

**Comment by stephansainleger** [comment]
version 1.0.0.10 - I do not reproduce the issue. Do you still have it @MatSarr ?
